app/common/history.ts

```diff
@@ -0,0 +1,49 @@
+export interface UpdateStyleAction {
+  type: "update-style";
+  targets: Array<{ webviewId: string; selector: string }>;
+  selector: string;
+  style: string;
```

Comment:
I would follow the pattern from this PR https://github.com/onlook-dev/monorepo/blob/main/editor/src/lib/tools/edit/history.ts, making an action for each "kind" of UI action. Obvs the boundaries are fuzzy here, but as a general principle instead of "superactions" that are the general case of many specific actions, I'd go with small and simple actions that are straightforward to understand in isolation, instantiate at a call site and reverse on undo